web/app/test/ServiceMeshTest.jsx

```diff
@@ -38,59 +39,147 @@ describe('ServiceMesh', () => {
-      expect(component.html()).to.include("deployments have not been added to the service mesh.");
+      expect(component.find("ServiceMesh")).to.have.length(1);
+      expect(component.find("ConduitSpinner")).to.have.length(0);
+      expect(component.find("CallToAction")).to.have.length(1);
     });
   });
 
-  it("displays message for only one deployment not added to servicemesh", () => {
-    let addedPods = _.clone(podFixtures.pods);
+  it("renders controller component summaries", () => {
+    let addedPods = _.cloneDeep(podFixtures.pods);
     _.set(addedPods[0], "added", true);
 
     fetchStub.returnsPromise().resolves({
       ok: true,
-      json: () => Promise.resolve({ pods: addedPods})
+      json: () => Promise.resolve({ metrics: multiDeployRollupFixtures.metrics, pods: addedPods})
     });
     component = mount(routerWrap(ServiceMesh));
 
     return withPromise(() => {
-      expect(component.html()).to.include("1 deployment has not been added to the service mesh.");
+      expect(component.find("ServiceMesh")).to.have.length(1);
+      expect(component.find("ConduitSpinner")).to.have.length(0);
+      expect(component.find("DeploymentSummary")).to.have.length(3);
     });
   });
 
-  it("displays message for all deployments added to servicemesh", () => {
-    let addedPods = _.clone(podFixtures.pods);
-    _.forEach(addedPods, pod => {
-      _.set(pod, "added", true);
+  it("renders service mesh details section", () => {
+    fetchStub.returnsPromise().resolves({
+      ok: true,
+      json: () => Promise.resolve({ metrics: [] })
+    });
+    component = mount(routerWrap(ServiceMesh));
+
+    return withPromise(() => {
+      expect(component.find("ServiceMesh")).to.have.length(1);
+      expect(component.find("ConduitSpinner")).to.have.length(0);
+      expect(component.html()).includes("Service mesh details");
+      expect(component.html()).includes("Conduit version");
+    });
+  });
+
+  it("renders control plane section", () => {
+    fetchStub.returnsPromise().resolves({
+      ok: true,
+      json: () => Promise.resolve({ metrics: [] })
+    });
+    component = mount(routerWrap(ServiceMesh));
+
+    return withPromise(() => {
+      expect(component.find("ServiceMesh")).to.have.length(1);
+      expect(component.find("ConduitSpinner")).to.have.length(0);
+      expect(component.html()).includes("Control plane");
     });
+  });
 
+  it("renders data plane section", () => {
     fetchStub.returnsPromise().resolves({
       ok: true,
-      json: () => Promise.resolve({ pods: addedPods})
+      json: () => Promise.resolve({ metrics: [] })
     });
     component = mount(routerWrap(ServiceMesh));
 
     return withPromise(() => {
-      expect(component.html()).to.include("All deployments have been added to the service mesh.");
+      expect(component.find("ServiceMesh")).to.have.length(1);
+      expect(component.find("ConduitSpinner")).to.have.length(0);
+      expect(component.html()).includes("Data plane");
```

Comment:
let's also standardize `.includes` and `.to.include`
(I think these read better as "expect <blah> to include <blah>")